open-build-service/.gitignore

```diff
@@ -0,0 +1,6 @@
+notqmail.spec
```

Comment:
Can we add a comment explaining what to keep this synchronized with? (And a matching comment on the other end.)

Reply:
notqmail.spec is generated from notqmail.spec.in in the Makefile. There are two files that need to be maintained

1. conf-version - This is the notqmail version. e.g. 1.09
2. conf-release - This is the release number for the binary package. e.g. first build could be 1.1

I will update notqmail.spec and other file notqmail.dsc.in, that are generated using rules in the Makefile. Future version could pick this information from the ChangeLog / Release Notes